### PR TITLE
add welcome page and login page

### PR DESCRIPTION
**Pull request by sashameniakin**
no login function, just link to home page
welcome page -> login page -> home page

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+import {useRouter} from "next/router";
+
 function MyApp({Component, pageProps}) {
-  return (
-    <>
-      <GlobalStyles />
-      <Layout>
+  const router = useRouter();
+  const {asPath} = router;
+  console.log(asPath);
```
Comment (stefankorder) [inline pages/_app.js:9]: Please delete this console log. There should be no console log in your final PR.

File: pages/_app.js [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
 
+import {useRouter} from "next/router";
+
 function MyApp({Component, pageProps}) {
-  return (
-    <>
-      <GlobalStyles />
-      <Layout>
+  const router = useRouter();
+  const {asPath} = router;
```
Comment (stefankorder) [inline pages/_app.js:8]: Maybe you can write this like: 
```suggestion
  const {asPath}  = useRouter();
```

File: pages/_app.js [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+    return (
+      <>
+        <GlobalStyles />
 
+        <Layout>
+          <Component {...pageProps} />
+        </Layout>
+      </>
+    );
+  }
```
Comment (stefankorder) [inline pages/_app.js:29]: There could be a better way to display your components:
```
return (
      <>
        <GlobalStyles />
           {asPath === "/login" || asPath === "/" ? 
           <Component {...pageProps} /> : 
           <Layout>
             <Component {...pageProps} />
           </Layout> }
      </>
    );
```

File: components/Navbar.js [hunk 4]
```diff
@@ -17,8 +17,8 @@ export default function Navbar() {
 
       <footer>
         <StyledList>
-          <StyledDiv active={pathname === "/" ? true : false}>
-            <StyledLink href="/">
+          <StyledDiv active={pathname === "/home" ? true : false}>
```
Comment (stefankorder) [inline components/Navbar.js:20]: Do you use the StyledDiv component only to style the StyledLink component?

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Please take a look at my comments.

File: pages/_app.js (hunk 1 above)
Comment (sashameniakin) [reply pages/_app.js:9]: done!

File: pages/_app.js (hunk 2 above)
Comment (sashameniakin) [reply pages/_app.js:8]: done!

File: components/Navbar.js (hunk 4 above)
Comment (sashameniakin) [reply components/Navbar.js:20]: I use the StyledDiv to highlight the active page icon

**Review by actyralf — APPROVED** [review]
Well done!

**Pull request merged**